docs/verification-integration.md

```diff
@@ -0,0 +1,108 @@
+# Verification system
+
+Applications are executed in our Verification system to ensure that:
+
+1. Installation succeeds: All resources are applied and waited for to become healthy.
+1. Functionality tests pass: The deployer starts the Tester Pod and watches its exit status. Zero means
+success, non-zero means failure.
+1. Uninstallation succeeds: Application and all its resources are successfully removed from the cluster.
+
+Successful results are required before application can be published to the GCP Marketplace.
+
+## Functionality tests
+
+Packaging functionality tests involves the following steps.
+
+### 1. Create a `/data-test` folder
+
+The structure is similar to the one in `/data` folder, but with the configuration and manifests that
+are only meant to be applied when the application is installed in verification mode. Some examples of
+`/data-test` for different deployers:
```

Comment:
Is there a special deployer for K8s Operators?
If yes, when do you plan to add it, especially, since we plan to bring ~80 K8s operators from https://operatorhub.io to GCP MP's in Q4?

Reply:
There is no special deployer for operators as far as I know. This doc only covers the existing deployers.

Reply:
Should we create a new bug for Operators then?

Here are the partners that have operators, some of them have listings, some are working on getting them on MP and Anthos compatible:

- Confluent
- Elastic (it's in Alpha)
- MongoDB
- Couchbase
- TiDB
- Robin

The current plan is not pretty much make K8s Operator a standard and move as many partner to that format, plus bring new ones too.